c++/src/capnp/compat/http-over-capnp-test.c++

```diff
@@ -960,14 +990,14 @@ KJ_TEST("HTTP-over-Cap'n-Proto Connect with startTls") {
   auto request = frontCapnpHttpClient->connect(
       "https://example.org"_kj, kj::HttpHeaders(*table), settings);
 
-  KJ_ASSERT_NONNULL(*tlsStarter);
+  auto& tlsStarterCallback = KJ_ASSERT_NONNULL(*tlsStarter);
```

Comment:
This change is just to appease my editor's syntax highlighter.
The below line "`KJ_ASSERT_NONNULL(*tlsStarter)("example.com")`" caused the rest of the file to be treated as a string..

Reply:
Weird, what editor?

Reply:
vscode 😞 
I can revert it back if you dislike it, just assumed you wouldn't care much

Reply:
No it's fine, I was just curious.

I also use vscode and... wow, you're right, the highlight is all messed up after `("example.com")`